pf/internal/convert/bool.go

```diff
@@ -49,8 +49,7 @@ func (*boolEncoder) FromPropertyValue(p resource.PropertyValue) (tftypes.Value,
 
 func (*boolDecoder) ToPropertyValue(v tftypes.Value) (resource.PropertyValue, error) {
 	if !v.IsKnown() {
-		c := resource.Computed{Element: resource.NewBoolProperty(false)}
-		unknown := resource.NewComputedProperty(c)
+		unknown := resource.NewOutputProperty(resource.Output{Known: false})
```

Comment:
This could be cleaned up with a constant in the convert module:

```go
var unknownProperty = resource.NewOutputProperty(resource.Output{Known: false})
```